README.md

```diff
@@ -190,22 +207,26 @@ There is a simple demo provided, but because of Looker's attention to security,
 * Navigate to Admin > *Platform* Embed on your Looker instance. This requires Admin privileges.
 * The demo server runs by default at [http://localhost:8080](http://localhost:8080). By adding that address to "Embedded Domain Whitelist" you can enabled the demo to receive messages from Looker.
 * Turn on "Embed Authentication"
-* In order to view your "Embed Secret" you must reset it. Copy the secret to someplace secure.
+* In order to use embedding you must generate an "Embed Secret"
```

Comment:
Is this true, since you can also use the Looker API option instead, no? Or do we require an embed secret to exist in order for the API endpoint to work?

Reply:
Yea, IIUC you have to create an embed secret even you use the API

Reply:
LGTM